clients/js/src/compat.ts

```diff
@@ -247,6 +252,26 @@ export function wrapEthersProvider<P extends Provider | Ethers5Provider>(
     estimateGas: hookEthersCall(provider, 'estimateGas', filled_options),
   };
 
+  // Provide an EIP-1193 compatible endpoint for Ethers providers
+  if (!('request' in provider) && 'send' in provider) {
+    hooks['request'] = async function (args: {
+      method: string;
+      params: any[];
+    }) {
+      const cipher = await filled_options.fetcher.cipher(provider);
+      const { method, params } = await prepareRequest(
+        args,
+        cipher,
+        signer as any,
+      );
+      const res = await (provider as any).send(method, params ?? []);
```

Comment:
It doesn't solve this `any` instance, the `send` interface is exposed by `JsonRpcProvider` and `BrowserProvider`, but this could be from ether the Ethers 5 or Ethers 6 packages.